src/Interactive/EditorFeatures/Core/Extensibility/Interactive/CSharpVBResetCommand.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Language.StandardClassification;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Classification;
+using Microsoft.VisualStudio.InteractiveWindow;
+using Microsoft.VisualStudio.Utilities;
+using Microsoft.VisualStudio.Editor.Interactive;
+using Microsoft.CodeAnalysis.Editor.Interactive;
+
+namespace Microsoft.VisualStudio.InteractiveWindow.Commands
+{
+
+    /// <summary>
+    /// Represents a command which can be run from a REPL window.
+    /// 
+    /// This interface is a MEF contract and can be implemented and exported to add commands to the REPL window.
```

Comment:
The second sentence is a comment for the interface rather than the implementation.